ci-operator/templates/openshift/openshift-ansible/cluster-launch-e2e-40.yaml

```diff
@@ -255,34 +251,80 @@ objects:
         trap 'rc=$?; if test "${rc}" -eq 0; then touch /tmp/config-success; else touch /tmp/exit; fi; exit "${rc}"' EXIT
         trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN}; fi' TERM
 
-        if [[ ${TYPE} == 'gcp' ]]; then
-          export OPENSHIFT_INSTALL_PLATFORM=libvirt
-          export OPENSHIFT_INSTALL_BASE_DOMAIN=origin-ci-int-gce.dev.rhcloud.com
-          export OPENSHIFT_INSTALL_LIBVIRT_URI="qemu+tcp://192.168.122.1/system"
-          export OPENSHIFT_INSTALL_LIBVIRT_IMAGE="file:///unused"
-        fi
-        if [[ ${TYPE} == 'aws' ]]; then
-          export OPENSHIFT_INSTALL_PLATFORM="aws"
-          export OPENSHIFT_INSTALL_BASE_DOMAIN="test.ose"
-          export AWS_SHARED_CREDENTIALS_FILE="/etc/openshift-installer/.awscred"
-          export OPENSHIFT_INSTALL_AWS_REGION="us-east-1"
-        fi
         mkdir /tmp/artifacts/installer &&
         /bin/openshift-install version >/tmp/artifacts/installer/version
 
-        export _CI_ONLY_STAY_AWAY_OPENSHIFT_INSTALL_AWS_USER_TAGS="{\"expirationDate\": \"$(date -d '4 hours' --iso=minutes --utc)\"}"
-        /bin/openshift-install --dir=/tmp/artifacts/installer --log-level=debug create install-config
+        export CLUSTER_ID=$(uuidgen --random)
```

Comment:
Why not use `JOB_NAME_HASH`? This would help us identify which resources are created by particular CI job

Reply:
The cluster ID has to be a version 4 UUID. If `JOB_NAME_HASH` satisfies that constraint we can use it. Do you know if it does?

Reply:
Ah, right, it doesn't. Seems this value would be logged in the container logs, so resource can be tracked down to the CI job